SOCIAL-MEDIA.md

```diff
@@ -0,0 +1,102 @@
+- Engage with the community through reposts and sharing of community content.
+
+### General Guidelines
+
+For all CNCF social activity, we remain a neutral foundation, in compliance with [CNCF's vendor neutrality principles](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+Examples of the type of content:
+
+- Owned content is CNCF news, blogs, case studies, survey data, etc.
+- Project content is anything sourced from CNCF's currently hosted projects, including project news, roadmap updates, new releases, performance/security updates, blogs, conferences slides/videos, etc.
+- Ecosystem content is vendor-neutral and project-impartial sourced from contributors, maintainers, ambassadors, news outlets, etc., including blog posts, news coverage, thought leadership bylines, technology demos, sketch notes, GitHub work, cloud native-specific meetups, etc.
+- CNCF is not able to share anything on our channels that promotes a vendor product or directs to a company website.
+- Social posts from member company handles and/or vendor channels cannot be shared.
+- Reposts are limited to news outlet handles such as [@linuxfoundation](https://twitter.com/linuxfoundation), [@kubecon_](https://twitter.com/kubecon_), project handles, personal handles and their counterparts in other social media platforms.
+- Community content that abides by the channel guidelines + includes insight from/work with more than one CNCF project will be prioritized for sharing.
+- Activities that are hosted by and open to the public can be promoted, as they benefit the ecosystem as a whole.
+
+### Images & Videos
+
+Images shared, unless specifically credited back to a community member, will meet the requirements of "free for commercial use" and "no attribution required."
+
+### Hashtags
+
+CNCF and Knative uses hashtags in our posts:
+
+- To measure the success of campaigns (for example, #KubeCon).
+- To expand our reach beyond our current followers and tap into larger, trending topics on Twitter (for example, #cloudnative).
+- To organize or categorize shares (for example, #Kubernetes).
+- Add #knative hashtag to posts if possible.
+
+### To request content to be shared
+
+Share your social media post on the [#knative-socialmedia](https://cloud-native.slack.com/archives/C05R3575M9U) channel on the CNCF Slack.
+
+## Regular posts
+
+### New contributor celebration
+
+Every month, we celebrate new contributors to the Knative project. We will share a post on Twitter and LinkedIn to celebrate the new contributors.
+See the [Knative contributor data](https://github.com/aliok/knative-contrib-report?tab=readme-ov-file#list-of-all-contributors-to-the-knative-project-2) for more information on how to find new contributors.
+
+### Biweekly social media post
+
+After each Knative Steering Committee meeting, we will share a post on Twitter and LinkedIn to share the highlights of the past 2 weeks.
```

Comment:
This seems like it might veer into the "announcements" or "news" aspect.  Other that that, this LGTM.